odoo/addons/base/models/res_lang.py

```diff
@@ -277,10 +277,13 @@ def write(self, vals):
         if 'code' in vals and any(code != vals['code'] for code in lang_codes):
             raise UserError(_("Language code cannot be modified."))
         if vals.get('active') == False:
+            inactive_users_lang = self.env['res.users'].search([('active', '=', False)]).mapped('lang')
```

Comment:
This call is very inefficient on a database with many users, like _our own_ database.